### ISSUE TITLE
Does not compatible with macOS 10.12

### ISSUE THREAD

**Issue by SharkIng** [digitalocean/doctl#128]
```
Warning: You are using OS X 10.12.
We do not provide support for this pre-release version.
You may encounter build failures or other breakages.
Please create pull-requests instead of filing issues.
==> Upgrading 1 outdated package, with result:
doctl 1.4.0
go: This formula either does not compile or function as expected on OS X
versions newer than El Capitan due to an upstream incompatibility.
Error: An unsatisfied requirement failed this build.
```

**Comment by bryanl** [comment]
I don't have 10.12 yet, but it looks like there is a bigger issue with compiling go. I will try to confirm.

**Comment by bmamlin** [comment]
FWIW, I am on macOS 10.12.1, ran `brew install doctl` and `doctl auth init`, then was able to list my droplets, create droplets, ssh into droplets, and delete droplets without any problems. I haven't tested all the commands.

```bash
$ doctl version
doctl version 1.5.0-release
```

**Comment by xmudrii** [comment]
@bmamlin It was fixed in 1.5.0. Make sure you updated doctl (you should be able to check version with `doctl version`). If you still experience problems, fire up a new issue.

There was problem with 10.12 and Go, #149 was also problem of this. And as said in one of comment version compiled with 1.7.1 (doctl 1.5.0) should fix problems.

I think it's ok to close this, cc @hugocorbucci @nanzhong @mchitten

**Comment by nanzhong** [comment]
👍 Agreed, closing this issue.
